src/cloud.js

```diff
@@ -28,10 +28,12 @@
 */
 
 // Global settings /////////////////////////////////////////////////////
+// cloud.js should be able to exist indepent of Snap!
```

Comment:
cloud.js exists for more than just Snap<em>!</em> now. 

I'm starting to think this file might belong in it's own repo (like morphic).